core/new-gui/src/app/dashboard/service/workflow-executions/workflow-executions.service.ts

```diff
@@ -41,4 +57,37 @@ export class WorkflowExecutionsService {
       executionName,
     });
   }
+
+  public displayWorkflowExecution(wid: number, vid: number): Observable<Workflow> {
+    return this.http.get<Workflow>(`${WORKFLOW_VERSIONS_API_BASE_URL}/${wid}/${vid}`).pipe(
+      filter((workflow: Workflow) => workflow != null),
+      map(WorkflowUtilService.parseWorkflowInfo)
+    );
+  }
+
+  public getDisplayParticularExecutionStream(): Observable<boolean> {
+    return this.displayParticularWorkflowExecution.asObservable();
+  }
+
+  public setDisplayParticularExecution(flag: boolean): void {
+    this.displayParticularWorkflowExecution.next(flag);
+  }
+
+  public closeParticularExecutionDisplay(): void {
```

Comment:
I don't think you need any of these, unlike version display where the user was on the workflowComponent and had to swap the graphs to display a temporarily view, the execution didn't have a temporarily graph and no need to do any of these swapping operations

Reply:
removed reset canvas code but remain the redirection to execution table code